frontend/src/Types/enums.ts

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
I found a cheatsheet how to name enum in TS
https://basarat.gitbook.io/typescript/styleguide#enum